test/unit/liquid/drops/account_drop_test.rb

```diff
@@ -22,7 +22,7 @@ def test_url_helpers
   end
 
   test 'returns vat_rate' do
-    assert_equal(@drop.vat_rate, @buyer.vat_rate)
+    assert_nil(nil, @buyer.vat_rate)
```

Comment:
This doesn't seem right.

Why are `event.provider` and `@drop.vat_rate` `nil` though?

Reply:
@akostadinov  This always comes nil when we run the test and  so we receive this warning .

 @drop is **Liquid::Drops::Account** not any object and its vat rate is always nil
@buyer is buyers account and to this account also vat rate is nil 
so basically whoever added this test is checking nil to  nil and so it gives a warning

there is no new change with the data of the object they are as it is .
I don't know why someone was checking nil equal values, so this fix actually clear that they are checking nil

Reply:
This warning  was keep showing when test are run so I fixed the warning

Reply:
The issues are two:
1. `assert_nil(nil, @buyer.vat_rate)` is not correct parameters for the assertion
2. the test is called `returns vat_rate`, if it is always `nil`, the test name seems confusing

Reply:
yes, create object with VAT. Also the `nil` check is good to be performed, just fix the parameters.